google/cloud/firestore_v1/base_aggregation.py

```diff
@@ -230,7 +237,12 @@ def get(
             retries.Retry, None, gapic_v1.method._MethodDefault
         ] = gapic_v1.method.DEFAULT,
         timeout: float | None = None,
-    ) -> List[AggregationResult] | Coroutine[Any, Any, List[AggregationResult]]:
+        *,
+        explain_options: Optional[ExplainOptions] = None,
+    ) -> (
+        QueryResultsList[AggregationResult]
+        | Awaitable[Any, Any, List[AggregationResult]]
```

Comment:
I think Awaitable only takes one argument. And I think it should be QueryResultsList, not just List